compiler/src/locals.rs

```diff
@@ -23,20 +33,36 @@ impl LocalsLayout {
     ///
     /// # Panics
     /// Panics if the local id is out of bounds.
-    pub fn set_space(&mut self, id: LocalId, stack_size: ValueStackSize) {
+    pub fn set_value_space(&mut self, id: LocalId, stack_size: ValueStackSize) {
         let size: u8 = stack_size.into();
-        self.indexes[id.0] = Some(self.len);
+        self.values_indexes[id.0] = Some(self.len);
         self.len += size as u32;
     }
 
+    /// Reserves the space in the eternal's reference
+    ///
+    /// Different initialization orders will result in different indexes.
+    pub fn set_external_ref_space(&mut self, symbol: ResolvedSymbol) {
+        match self.external_refs_indexes.entry(symbol) {
+            Entry::Occupied(_) => {}
```

Comment:
Don't silence errors.

Reply:
it's not an error, if the index is already initialized for the given symbol, then no need to determine a new one. I'll refactor this to `init_external_ref_space` to be more explicit